test/new-e2e/system-probe/system-probe-test-env.go

```diff
@@ -35,9 +35,19 @@ const (
 	PrimaryAZ   = "subnet-03061a1647c63c3c3"
 	SecondaryAZ = "subnet-0f1ca3e929eb3fb8b"
 	BackupAZ    = "subnet-071213aedb0e1ae54"
+
+	SandboxPrimaryAz   = "subnet-b89e00e2"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should we rather define these in `test-infra` and import definitions here, likely done for config map keys ? We might miss changes in `test-infra` by redefining them here.

Also would give more explicit names for `PrimaryAZ-BackupAZ` up at line 35-37. What account are they defined on ?

Reply:
We have a specific order in which we want to try the AZs, based on some experimentation to determine the availability of the particular instance types we target.
We want to override the whatever preference order is defined in `test-infra` so we redefine them here.